### ISSUE TITLE
Duplicates of jruby-core and jruby-stdlib in war file

### ISSUE THREAD

**Issue by colindean** [jruby/warbler#171]
This is my first time trying to ship a war file as a completely standalone web application. 

I'm seeing duplicate copies of jruby jars in my generated war file:

```
13865096  07-22-13 11:10   WEB-INF/lib/gems-gems-jruby-jars-1.7.4-lib-jruby-core-1.7.4.jar
  8091407  07-22-13 11:10   WEB-INF/lib/gems-gems-jruby-jars-1.7.4-lib-jruby-stdlib-1.7.4.jar
   243307  07-22-13 11:10   WEB-INF/lib/gems-gems-jruby-rack-1.1.13.2-lib-jruby-rack-1.1.13.2.jar
    16476  07-22-13 11:10   WEB-INF/lib/gems-gems-warbler-1.3.8-lib-warbler_jar.jar
  1947442  07-22-13 11:10   WEB-INF/lib/gems.jar
 <java jar dependencies snipped>
     7522  07-22-13 11:10   WEB-INF/lib/include_list.class
       35  07-22-13 11:10   WEB-INF/lib/include_list.rb
 13865096  07-22-13 11:10   WEB-INF/lib/jruby-core-1.7.4.jar
   243307  07-22-13 11:10   WEB-INF/lib/jruby-rack-1.1.13.2.jar
  8091407  07-22-13 11:10   WEB-INF/lib/jruby-stdlib-1.7.4.jar
```

My config/warble.rb:

```
# Warbler web application assembly configuration file
Warbler::Config.new do |config|
  config.features += ['executable','compiled','gemjar']
end
```

It seems that 

jruby-{core,rack,stdlib} are being included twice: once in `lib` and once as `lib/gems-gems-$name`. 

Any idea what could be going on?

**Comment by ari** [comment]
Is there any progress on this issue? How are we to avoid duplicate jars in our war? I'm getting duplicates of:

* jurby-rack
* jdbc-postgresql
* jruby-stdlib
* jruby-core

I'm using this in my gemfile

    gem "jruby-jars", "1.7.22"

Obviously I need to specify a specific version here. How can we avoid this warbler bug?

**Comment by kares** [comment]
has been fixed for warbler >= 2.0.0
